src/components/Navbar/index.module.scss

```diff
@@ -76,3 +76,15 @@
   font-size: 20px;
   color: #707070;
 }
+
+.InlineMenu {
```

Comment:
this doesn't look like the correct solution, you're just overriding the issue, go find what is causing it to be dark